pytorch_lightning/loggers/comet.py

```diff
@@ -123,6 +126,7 @@ def __init__(self,
         self.workspace = workspace
         self.project_name = project_name
         self.experiment_key = experiment_key
+        self.experiment_name = experiment_name
```

Comment:
we should make these protected, since they could be inconsisten with what the experiment object saves, right? Besides, since you have a setter for name that sets the project name, at least self.project_name should become self._project_name.